[2024-07-31T18:43:29.672+0000] {processor.py:161} INFO - Started process (PID=3490) to work on /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T18:43:29.673+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-07-31T18:43:29.675+0000] {logging_mixin.py:188} INFO - [2024-07-31T18:43:29.674+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T18:43:29.678+0000] {logging_mixin.py:188} INFO - [2024-07-31T18:43:29.677+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-07-31T18:43:29.678+0000] {logging_mixin.py:188} INFO - [2024-07-31T18:43:29.678+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-07-31T18:43:29.679+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T18:43:29.690+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.024 seconds
[2024-07-31T18:44:23.488+0000] {processor.py:161} INFO - Started process (PID=3545) to work on /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T18:44:23.489+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-07-31T18:44:23.491+0000] {logging_mixin.py:188} INFO - [2024-07-31T18:44:23.491+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T18:44:23.494+0000] {logging_mixin.py:188} INFO - [2024-07-31T18:44:23.494+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-07-31T18:44:23.495+0000] {logging_mixin.py:188} INFO - [2024-07-31T18:44:23.495+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-07-31T18:44:23.495+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T18:44:23.506+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.024 seconds
[2024-07-31T18:45:13.876+0000] {processor.py:161} INFO - Started process (PID=3603) to work on /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T18:45:13.877+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-07-31T18:45:13.879+0000] {logging_mixin.py:188} INFO - [2024-07-31T18:45:13.879+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T18:45:13.882+0000] {logging_mixin.py:188} INFO - [2024-07-31T18:45:13.882+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-07-31T18:45:13.883+0000] {logging_mixin.py:188} INFO - [2024-07-31T18:45:13.883+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-07-31T18:45:13.883+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T18:45:13.894+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.024 seconds
[2024-07-31T18:46:17.288+0000] {processor.py:161} INFO - Started process (PID=3671) to work on /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T18:46:17.290+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-07-31T18:46:17.292+0000] {logging_mixin.py:188} INFO - [2024-07-31T18:46:17.291+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T18:46:17.295+0000] {logging_mixin.py:188} INFO - [2024-07-31T18:46:17.294+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-07-31T18:46:17.295+0000] {logging_mixin.py:188} INFO - [2024-07-31T18:46:17.295+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-07-31T18:46:17.295+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T18:46:17.306+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.024 seconds
[2024-07-31T18:47:14.893+0000] {processor.py:161} INFO - Started process (PID=3925) to work on /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T18:47:14.894+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-07-31T18:47:14.896+0000] {logging_mixin.py:188} INFO - [2024-07-31T18:47:14.895+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T18:47:14.899+0000] {logging_mixin.py:188} INFO - [2024-07-31T18:47:14.898+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-07-31T18:47:14.899+0000] {logging_mixin.py:188} INFO - [2024-07-31T18:47:14.899+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-07-31T18:47:14.899+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T18:47:14.910+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.023 seconds
[2024-07-31T18:48:11.104+0000] {processor.py:161} INFO - Started process (PID=4155) to work on /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T18:48:11.106+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-07-31T18:48:11.107+0000] {logging_mixin.py:188} INFO - [2024-07-31T18:48:11.107+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T18:48:11.111+0000] {logging_mixin.py:188} INFO - [2024-07-31T18:48:11.110+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-07-31T18:48:11.111+0000] {logging_mixin.py:188} INFO - [2024-07-31T18:48:11.111+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-07-31T18:48:11.111+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T18:48:11.122+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.023 seconds
[2024-07-31T18:49:11.342+0000] {processor.py:161} INFO - Started process (PID=4216) to work on /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T18:49:11.344+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-07-31T18:49:11.345+0000] {logging_mixin.py:188} INFO - [2024-07-31T18:49:11.345+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T18:49:11.349+0000] {logging_mixin.py:188} INFO - [2024-07-31T18:49:11.348+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-07-31T18:49:11.349+0000] {logging_mixin.py:188} INFO - [2024-07-31T18:49:11.349+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-07-31T18:49:11.349+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T18:49:11.360+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.024 seconds
[2024-07-31T18:50:10.599+0000] {processor.py:161} INFO - Started process (PID=4280) to work on /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T18:50:10.600+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-07-31T18:50:10.602+0000] {logging_mixin.py:188} INFO - [2024-07-31T18:50:10.601+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T18:50:10.605+0000] {logging_mixin.py:188} INFO - [2024-07-31T18:50:10.605+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-07-31T18:50:10.606+0000] {logging_mixin.py:188} INFO - [2024-07-31T18:50:10.605+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-07-31T18:50:10.606+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T18:50:10.617+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.024 seconds
[2024-07-31T18:51:10.678+0000] {processor.py:161} INFO - Started process (PID=4347) to work on /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T18:51:10.679+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-07-31T18:51:10.681+0000] {logging_mixin.py:188} INFO - [2024-07-31T18:51:10.681+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T18:51:10.684+0000] {logging_mixin.py:188} INFO - [2024-07-31T18:51:10.684+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-07-31T18:51:10.685+0000] {logging_mixin.py:188} INFO - [2024-07-31T18:51:10.685+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-07-31T18:51:10.685+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T18:51:10.695+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.023 seconds
[2024-07-31T18:52:07.595+0000] {processor.py:161} INFO - Started process (PID=4408) to work on /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T18:52:07.597+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-07-31T18:52:07.599+0000] {logging_mixin.py:188} INFO - [2024-07-31T18:52:07.598+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T18:52:07.602+0000] {logging_mixin.py:188} INFO - [2024-07-31T18:52:07.601+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-07-31T18:52:07.602+0000] {logging_mixin.py:188} INFO - [2024-07-31T18:52:07.602+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-07-31T18:52:07.603+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T18:52:07.613+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.024 seconds
[2024-07-31T18:53:07.474+0000] {processor.py:161} INFO - Started process (PID=4473) to work on /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T18:53:07.475+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-07-31T18:53:07.476+0000] {logging_mixin.py:188} INFO - [2024-07-31T18:53:07.476+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T18:53:07.479+0000] {logging_mixin.py:188} INFO - [2024-07-31T18:53:07.478+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-07-31T18:53:07.480+0000] {logging_mixin.py:188} INFO - [2024-07-31T18:53:07.479+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-07-31T18:53:07.480+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T18:53:07.490+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2024-07-31T18:54:09.494+0000] {processor.py:161} INFO - Started process (PID=4542) to work on /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T18:54:09.496+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-07-31T18:54:09.497+0000] {logging_mixin.py:188} INFO - [2024-07-31T18:54:09.496+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T18:54:09.500+0000] {logging_mixin.py:188} INFO - [2024-07-31T18:54:09.500+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-07-31T18:54:09.501+0000] {logging_mixin.py:188} INFO - [2024-07-31T18:54:09.501+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-07-31T18:54:09.501+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T18:54:09.512+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.024 seconds
[2024-07-31T18:55:10.115+0000] {processor.py:161} INFO - Started process (PID=4611) to work on /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T18:55:10.117+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-07-31T18:55:10.118+0000] {logging_mixin.py:188} INFO - [2024-07-31T18:55:10.117+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T18:55:10.121+0000] {logging_mixin.py:188} INFO - [2024-07-31T18:55:10.120+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-07-31T18:55:10.121+0000] {logging_mixin.py:188} INFO - [2024-07-31T18:55:10.121+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-07-31T18:55:10.122+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T18:55:10.132+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2024-07-31T18:56:13.976+0000] {processor.py:161} INFO - Started process (PID=4678) to work on /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T18:56:13.978+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-07-31T18:56:13.978+0000] {logging_mixin.py:188} INFO - [2024-07-31T18:56:13.978+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T18:56:13.981+0000] {logging_mixin.py:188} INFO - [2024-07-31T18:56:13.981+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-07-31T18:56:13.982+0000] {logging_mixin.py:188} INFO - [2024-07-31T18:56:13.982+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-07-31T18:56:13.982+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T18:56:13.993+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2024-07-31T18:57:05.109+0000] {processor.py:161} INFO - Started process (PID=5044) to work on /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T18:57:05.111+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-07-31T18:57:05.112+0000] {logging_mixin.py:188} INFO - [2024-07-31T18:57:05.111+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T18:57:05.115+0000] {logging_mixin.py:188} INFO - [2024-07-31T18:57:05.114+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-07-31T18:57:05.115+0000] {logging_mixin.py:188} INFO - [2024-07-31T18:57:05.115+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-07-31T18:57:05.115+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T18:57:05.126+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2024-07-31T18:58:13.039+0000] {processor.py:161} INFO - Started process (PID=5114) to work on /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T18:58:13.041+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-07-31T18:58:13.042+0000] {logging_mixin.py:188} INFO - [2024-07-31T18:58:13.041+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T18:58:13.045+0000] {logging_mixin.py:188} INFO - [2024-07-31T18:58:13.044+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-07-31T18:58:13.045+0000] {logging_mixin.py:188} INFO - [2024-07-31T18:58:13.045+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-07-31T18:58:13.046+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T18:58:13.056+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2024-07-31T18:59:16.223+0000] {processor.py:161} INFO - Started process (PID=5185) to work on /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T18:59:16.224+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-07-31T18:59:16.225+0000] {logging_mixin.py:188} INFO - [2024-07-31T18:59:16.225+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T18:59:16.228+0000] {logging_mixin.py:188} INFO - [2024-07-31T18:59:16.228+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-07-31T18:59:16.229+0000] {logging_mixin.py:188} INFO - [2024-07-31T18:59:16.229+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-07-31T18:59:16.229+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T18:59:16.239+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2024-07-31T19:00:22.941+0000] {processor.py:161} INFO - Started process (PID=5256) to work on /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T19:00:22.942+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-07-31T19:00:22.943+0000] {logging_mixin.py:188} INFO - [2024-07-31T19:00:22.943+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T19:00:22.946+0000] {logging_mixin.py:188} INFO - [2024-07-31T19:00:22.946+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-07-31T19:00:22.947+0000] {logging_mixin.py:188} INFO - [2024-07-31T19:00:22.947+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-07-31T19:00:22.947+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T19:00:22.957+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.023 seconds
[2024-07-31T19:01:29.790+0000] {processor.py:161} INFO - Started process (PID=5325) to work on /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T19:01:29.794+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-07-31T19:01:29.795+0000] {logging_mixin.py:188} INFO - [2024-07-31T19:01:29.794+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T19:01:29.798+0000] {logging_mixin.py:188} INFO - [2024-07-31T19:01:29.797+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-07-31T19:01:29.798+0000] {logging_mixin.py:188} INFO - [2024-07-31T19:01:29.798+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-07-31T19:01:29.799+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T19:01:29.809+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.026 seconds
[2024-07-31T19:02:37.199+0000] {processor.py:161} INFO - Started process (PID=5394) to work on /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T19:02:37.205+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-07-31T19:02:37.206+0000] {logging_mixin.py:188} INFO - [2024-07-31T19:02:37.206+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T19:02:37.209+0000] {logging_mixin.py:188} INFO - [2024-07-31T19:02:37.209+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-07-31T19:02:37.210+0000] {logging_mixin.py:188} INFO - [2024-07-31T19:02:37.210+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-07-31T19:02:37.210+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T19:02:37.220+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.027 seconds
[2024-07-31T19:03:44.114+0000] {processor.py:161} INFO - Started process (PID=5464) to work on /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T19:03:44.116+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-07-31T19:03:44.117+0000] {logging_mixin.py:188} INFO - [2024-07-31T19:03:44.116+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T19:03:44.120+0000] {logging_mixin.py:188} INFO - [2024-07-31T19:03:44.119+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-07-31T19:03:44.120+0000] {logging_mixin.py:188} INFO - [2024-07-31T19:03:44.120+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-07-31T19:03:44.121+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T19:03:44.132+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.023 seconds
[2024-07-31T19:19:14.474+0000] {processor.py:161} INFO - Started process (PID=5632) to work on /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T19:19:14.476+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-07-31T19:19:14.477+0000] {logging_mixin.py:188} INFO - [2024-07-31T19:19:14.477+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T19:19:14.481+0000] {logging_mixin.py:188} INFO - [2024-07-31T19:19:14.480+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-07-31T19:19:14.481+0000] {logging_mixin.py:188} INFO - [2024-07-31T19:19:14.481+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-07-31T19:19:14.481+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T19:19:14.492+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.023 seconds
[2024-07-31T19:20:12.115+0000] {processor.py:161} INFO - Started process (PID=5688) to work on /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T19:20:12.116+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-07-31T19:20:12.118+0000] {logging_mixin.py:188} INFO - [2024-07-31T19:20:12.118+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T19:20:12.121+0000] {logging_mixin.py:188} INFO - [2024-07-31T19:20:12.121+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-07-31T19:20:12.122+0000] {logging_mixin.py:188} INFO - [2024-07-31T19:20:12.122+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-07-31T19:20:12.122+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T19:20:12.132+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.023 seconds
[2024-07-31T19:21:06.881+0000] {processor.py:161} INFO - Started process (PID=5744) to work on /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T19:21:06.882+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-07-31T19:21:06.884+0000] {logging_mixin.py:188} INFO - [2024-07-31T19:21:06.883+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T19:21:06.887+0000] {logging_mixin.py:188} INFO - [2024-07-31T19:21:06.886+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-07-31T19:21:06.887+0000] {logging_mixin.py:188} INFO - [2024-07-31T19:21:06.887+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-07-31T19:21:06.888+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T19:21:06.898+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.023 seconds
[2024-07-31T19:21:59.834+0000] {processor.py:161} INFO - Started process (PID=5798) to work on /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T19:21:59.836+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-07-31T19:21:59.838+0000] {logging_mixin.py:188} INFO - [2024-07-31T19:21:59.837+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T19:21:59.841+0000] {logging_mixin.py:188} INFO - [2024-07-31T19:21:59.840+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-07-31T19:21:59.841+0000] {logging_mixin.py:188} INFO - [2024-07-31T19:21:59.841+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-07-31T19:21:59.842+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T19:21:59.852+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.023 seconds
[2024-07-31T19:22:46.667+0000] {processor.py:161} INFO - Started process (PID=5846) to work on /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T19:22:46.669+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-07-31T19:22:46.670+0000] {logging_mixin.py:188} INFO - [2024-07-31T19:22:46.669+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T19:22:46.673+0000] {logging_mixin.py:188} INFO - [2024-07-31T19:22:46.672+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-07-31T19:22:46.673+0000] {logging_mixin.py:188} INFO - [2024-07-31T19:22:46.673+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-07-31T19:22:46.674+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T19:22:46.684+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.023 seconds
[2024-07-31T19:23:56.611+0000] {processor.py:161} INFO - Started process (PID=5922) to work on /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T19:23:56.612+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-07-31T19:23:56.613+0000] {logging_mixin.py:188} INFO - [2024-07-31T19:23:56.613+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T19:23:56.617+0000] {logging_mixin.py:188} INFO - [2024-07-31T19:23:56.616+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-07-31T19:23:56.617+0000] {logging_mixin.py:188} INFO - [2024-07-31T19:23:56.617+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-07-31T19:23:56.617+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T19:23:56.628+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2024-07-31T19:24:43.954+0000] {processor.py:161} INFO - Started process (PID=5974) to work on /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T19:24:43.955+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-07-31T19:24:43.956+0000] {logging_mixin.py:188} INFO - [2024-07-31T19:24:43.956+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T19:24:43.959+0000] {logging_mixin.py:188} INFO - [2024-07-31T19:24:43.959+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-07-31T19:24:43.960+0000] {logging_mixin.py:188} INFO - [2024-07-31T19:24:43.959+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-07-31T19:24:43.960+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T19:24:43.971+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.023 seconds
[2024-07-31T19:25:29.992+0000] {processor.py:161} INFO - Started process (PID=6024) to work on /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T19:25:29.994+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-07-31T19:25:29.995+0000] {logging_mixin.py:188} INFO - [2024-07-31T19:25:29.994+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T19:25:29.998+0000] {logging_mixin.py:188} INFO - [2024-07-31T19:25:29.997+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-07-31T19:25:29.998+0000] {logging_mixin.py:188} INFO - [2024-07-31T19:25:29.998+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-07-31T19:25:29.999+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T19:25:30.010+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.023 seconds
[2024-07-31T19:26:20.985+0000] {processor.py:161} INFO - Started process (PID=6078) to work on /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T19:26:20.986+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-07-31T19:26:20.987+0000] {logging_mixin.py:188} INFO - [2024-07-31T19:26:20.987+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T19:26:20.990+0000] {logging_mixin.py:188} INFO - [2024-07-31T19:26:20.990+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-07-31T19:26:20.991+0000] {logging_mixin.py:188} INFO - [2024-07-31T19:26:20.991+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-07-31T19:26:20.991+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T19:26:21.002+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.023 seconds
[2024-07-31T19:27:13.866+0000] {processor.py:161} INFO - Started process (PID=6136) to work on /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T19:27:13.868+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-07-31T19:27:13.869+0000] {logging_mixin.py:188} INFO - [2024-07-31T19:27:13.868+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T19:27:13.872+0000] {logging_mixin.py:188} INFO - [2024-07-31T19:27:13.871+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-07-31T19:27:13.872+0000] {logging_mixin.py:188} INFO - [2024-07-31T19:27:13.872+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-07-31T19:27:13.873+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T19:27:13.883+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.023 seconds
[2024-07-31T19:28:15.659+0000] {processor.py:161} INFO - Started process (PID=6200) to work on /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T19:28:15.660+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-07-31T19:28:15.661+0000] {logging_mixin.py:188} INFO - [2024-07-31T19:28:15.661+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T19:28:15.665+0000] {logging_mixin.py:188} INFO - [2024-07-31T19:28:15.664+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-07-31T19:28:15.665+0000] {logging_mixin.py:188} INFO - [2024-07-31T19:28:15.665+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-07-31T19:28:15.665+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T19:28:15.676+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2024-07-31T19:29:08.775+0000] {processor.py:161} INFO - Started process (PID=6254) to work on /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T19:29:08.777+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-07-31T19:29:08.778+0000] {logging_mixin.py:188} INFO - [2024-07-31T19:29:08.777+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T19:29:08.781+0000] {logging_mixin.py:188} INFO - [2024-07-31T19:29:08.780+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-07-31T19:29:08.781+0000] {logging_mixin.py:188} INFO - [2024-07-31T19:29:08.781+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-07-31T19:29:08.782+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T19:29:08.792+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.023 seconds
[2024-07-31T19:30:14.478+0000] {processor.py:161} INFO - Started process (PID=6330) to work on /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T19:30:14.479+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-07-31T19:30:14.480+0000] {logging_mixin.py:188} INFO - [2024-07-31T19:30:14.480+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T19:30:14.484+0000] {logging_mixin.py:188} INFO - [2024-07-31T19:30:14.483+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-07-31T19:30:14.484+0000] {logging_mixin.py:188} INFO - [2024-07-31T19:30:14.484+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-07-31T19:30:14.485+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T19:30:14.495+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.023 seconds
[2024-07-31T19:31:20.496+0000] {processor.py:161} INFO - Started process (PID=6399) to work on /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T19:31:20.497+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-07-31T19:31:20.498+0000] {logging_mixin.py:188} INFO - [2024-07-31T19:31:20.497+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T19:31:20.501+0000] {logging_mixin.py:188} INFO - [2024-07-31T19:31:20.500+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-07-31T19:31:20.501+0000] {logging_mixin.py:188} INFO - [2024-07-31T19:31:20.501+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-07-31T19:31:20.502+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T19:31:20.513+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.023 seconds
[2024-07-31T19:32:27.014+0000] {processor.py:161} INFO - Started process (PID=6471) to work on /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T19:32:27.015+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-07-31T19:32:27.016+0000] {logging_mixin.py:188} INFO - [2024-07-31T19:32:27.016+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T19:32:27.020+0000] {logging_mixin.py:188} INFO - [2024-07-31T19:32:27.019+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-07-31T19:32:27.020+0000] {logging_mixin.py:188} INFO - [2024-07-31T19:32:27.020+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-07-31T19:32:27.020+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T19:32:27.030+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2024-07-31T19:33:33.836+0000] {processor.py:161} INFO - Started process (PID=6544) to work on /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T19:33:33.838+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-07-31T19:33:33.839+0000] {logging_mixin.py:188} INFO - [2024-07-31T19:33:33.838+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T19:33:33.842+0000] {logging_mixin.py:188} INFO - [2024-07-31T19:33:33.841+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-07-31T19:33:33.842+0000] {logging_mixin.py:188} INFO - [2024-07-31T19:33:33.842+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-07-31T19:33:33.843+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T19:33:33.853+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2024-07-31T19:34:28.664+0000] {processor.py:161} INFO - Started process (PID=6601) to work on /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T19:34:28.665+0000] {processor.py:830} INFO - Processing file /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-07-31T19:34:28.666+0000] {logging_mixin.py:188} INFO - [2024-07-31T19:34:28.666+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T19:34:28.670+0000] {logging_mixin.py:188} INFO - [2024-07-31T19:34:28.669+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-07-31T19:34:28.670+0000] {logging_mixin.py:188} INFO - [2024-07-31T19:34:28.670+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-07-31T19:34:28.670+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-07-31T19:34:28.681+0000] {processor.py:183} INFO - Processing /home/ubuntu/airflow-venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.023 seconds
